examples/feeds/link_feed_item_set.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Links the specified feed item set to the specified feed item.
+
+The specified feed item set must not be created as a dynamic set, i.e. both
+dynamic_location_set_filter and dynamic_affiliate_location_set_filter must not
+be set. Learn more here:
+https://developers.google.com/google-ads/api/docs/location-extensions/feed-item-set?hl=en#adding-feed-item
+"""
+
+import argparse
+import sys
+
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+from google.ads.google_ads.util import ResourceName
+
+
+def main(client, customer_id, feed_id, feed_item_id, feed_item_set_id):
+    """The main method that creates all necessary entities for the example.
+
+    Args:
+        client: an initialized GoogleAdsClient instance.
+        customer_id: a client customer ID.
+        feed_id: the ID of the feed associated with the specified feed item set.
+        feed_item_id: the ID of the specified feed item.
+        feed_item_set_id: the ID of the feed item set to link to the feed item.
+    """
+    feed_item_set_link_service = client.get_service(
+        "FeedItemSetLinkService", version="v6"
+    )
+    feed_item_set_link_operation = client.get_type(
+        "FeedItemSetLinkOperation", version="v6"
+    )
+
+    # Construct an operation that will link the feed item to the feed item set.
+    feed_item_set_link = feed_item_set_link_operation.create
+
+    # Construct a resource name for a feed item set link, which is in the
+    # format: customers/{customer_id}/feedItemSetLinks/{feed_id}~{feed_item_set_id}~{feed_item_id}
```

Comment:
I think this comment is incorrect - this is the structure for the `FeedItemSetLink` object, but it's `feed_item` field is the resource name for a `FeedItem`, which is of the form `customers/{customer_id}/feedItems/{feed_id}~{feed_item_id}`

Reply:
Good catch, will fix

Reply:
done